requirements-moo.txt

```diff
@@ -0,0 +1 @@
+pymoo>=0.6.0
```

Comment:
We already have at least one multi-objective method in Syne Tune, right? If that's true, under what circumstances do customers need this dependency?

Reply:
I think we discussed this yesterday. The current MO methods in Syne Tune (mostly MO-ASHA) do not require dependencies beyond the core. But now, Jacek is owning this part, and he is introducing new tooling. I foresee there will be extra methods, also schedulers, which require these. For now, he only contributes a plotting functionality.